core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/JmxProxyFactoryImpl.java

```diff
@@ -303,14 +316,20 @@ public double getPercentRepaired(final TableReference tableReference)
                 return percentRepaired;
             }
             catch (AttributeNotFoundException
-                   | InstanceNotFoundException
                    | MBeanException
                    | ReflectionException
                    | IOException
                    | MalformedObjectNameException e)
             {
                 LOG.error("Unable to retrieve disk space usage for {}", tableReference, e);
             }
+            catch (RuntimeMBeanException
+                   | InstanceNotFoundException e)
+            {
+                // This exception can occur when a table is about to be changed, but has not finished doing so
+                // before we try to fetch data from it. Just return the default value for the time being.
+                LOG.warn("Unable to retrieve disk space usage for {} (bean not yet ready)", tableReference);
```

Comment:
"retrieve disk space usage"? Maybe something like "Percent Repaired in by incremental repair"